docs/src/examples.md

```diff
@@ -41,7 +41,59 @@ julia> round.(predict(ols), digits=5)
  6.83333
 ```
 
-.
+## Categorical variables
+
+Categorical variables will be dummy coded by default if they are non-numeric or if they are marked as categorical in an input DataFrame. Alternatively, you can pass an explicit [contrasts](https://juliastats.github.io/StatsModels.jl/latest/contrasts/) argument if you would like a different contrast coding system or if you are not using DataFrames.
```

Comment:
Actually any Table that stores the `CategoricalVector` would work, not just a dataframe.  For instance, a `ColumnTable` (e.g., named tuple of abstract vectors):

```
julia> lm(@formula(y ~ x), (y = rand(100), x = categorical(repeat(1:4, 25))))
StatsModels.TableRegressionModel{LinearModel{GLM.LmResp{Array{Float64,1}},GLM.DensePredChol{Float64,LinearAlgebra.Cholesky{Float64,Array{Float64,2}}}},Array{Float64,2}}

y ~ 1 + x

Coefficients:
──────────────────────────────────────────────────────────────────────────────
               Estimate  Std. Error    t value  Pr(>|t|)  Lower 95%  Upper 95%
──────────────────────────────────────────────────────────────────────────────
(Intercept)   0.518846    0.0562432   9.22505     <1e-14   0.407204  0.630488
x: 2         -0.0509322   0.0795399  -0.640335    0.5235  -0.208818  0.106953
x: 3          0.0485544   0.0795399   0.610441    0.5430  -0.109331  0.20644
x: 4         -0.0981986   0.0795399  -1.23458     0.2200  -0.256084  0.0596869
──────────────────────────────────────────────────────────────────────────────
```

Reply:
This is resolved in my new commit, I reckon